### ISSUE TITLE
Outdated i think

### ISSUE THREAD

**Issue by umairbajwa** [seleniumbase/SeleniumBase#169]
1. If you are to use tomcat user than in cloud launcher select Jenkins by Bitnami 
![screen shot 2018-07-03 at 3 22 08 pm](https://user-images.githubusercontent.com/38785717/42214395-e064ddfe-7ed4-11e8-8ac6-667df440ec2a.png)
2. There is no server_setup.py in repo: https://github.com/seleniumbase/SeleniumBase

**Comment by umairbajwa** [comment]
for google cloud integration readme file

**Comment by mdmintz** [comment]
I updated several ReadMe files. https://github.com/seleniumbase/SeleniumBase/commit/73520c67fe64f33f7529ae98d0afba0c309d8a8b . In the first thing you mentioned, looks like Google updated the name of the Jenkins item in Cloud Launcher. In the second thing you mentioned, I used to have a 2nd setup.py file called server_setup.py file for people who were still using selenium 2.53.6 instead of the latest official release for selenium 3. Let me know if you see any more issues. Thanks.
